classes/Cart.php

```diff
@@ -2002,14 +2002,18 @@ protected function _deleteCustomization($id_customization, $id_product, $id_prod
             WHERE `id_customization` = ' . (int) $id_customization);
 
         if ($customization) {
-            $cust_data = Db::getInstance()->getRow('SELECT *
+            $cust_datas = Db::getInstance()->executeS('SELECT *
```

Comment:
```suggestion
            /*
             * Now we will select all customized files that could be connected to this customization.
             * One customization can have multiple fields for files, we need to delete all of them.
             */
            $cust_datas = Db::getInstance()->executeS('SELECT *
```

Just some comments 🙏